### DISCUSSION TITLE
0.1.0-alpha.1

### DISCUSSION THREAD

**Discussion by kikobar** [FabLabSG/fablab-docs#6] [Announcements]
Note: This is a pre-release version.

This version is released for review by the team.

This version contains:

* Terms of Reference (ToR) for the whole project (README.md)
* Terms of Reference (ToR) for the first phase of the project (Conceptualisation ToR.md)

<hr /><em>This discussion was created from the release <a href='https://github.com/FabLabSG/fablab-docs/releases/tag/0.1.0-alpha.1'>0.1.0-alpha.1</a>.</em>

**Comment by makenplaystudiosg** [comment]
Besides the Challenge is important that the communities have a space to showcase projects and for potential clients to go and see what they have been done.  

Showcases could be linked to an already event happening, like harmony day, national day celebrations, Chingay parade, purple parade event or the sorts. 
Something the communities could be working year long  to prepare something meaningful to present.

**Reply by kikobar** [reply]
Correct, the challenges are just one form of organised engagement and the project's own social responsibility chapter, but the community is allowed to use the FabLab resources for whatever (legal) purpose they deem fit.

We would apply some transparent unit rates for every machine time and material so that anyone could pay for their own usage.
